### PR TITLE
Rockets_section

### PR DESCRIPTION
**Pull request by Ali-0111**
### Description:

- Store and slice created using `reduxjs/toolkit`, provided for react component
- `rocketslice.js` having initial state, actions created to make active state 
- rocket list rendered using state on UI
- reservation button and cancel reservation button work based on `reserved: true` attribute
- All components styled by separate `.css` files

### CODE REVIEW

File: src/Components/rockets/RocketsArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+              ? () => cancelHandler(articleData.id)
+              : () => reserveHandler(articleData.id)
+            }
+        >
+          {
+            articleData.reserved
+              ? 'Cancel Reservation'
+              : 'Reserve Rocket'
+          }
+        </button>
```
Comment (sameedmohsin28) [inline src/Components/rockets/RocketsArticle.jsx:54]: This piece of code does not follow the DRY principles and KISS principles to some extent too. Kindly try combining all three conditional blocks in a single conditional block.

**Review by sameedmohsin28 — CHANGES_REQUESTED** [review]
@Ali-0111

- [x] No Linter errors
- [x] Nice PR Description
- [x] Gitflow followed



While there is almost everything good in your code, a little fine-tuning would make it awesome.

**Comment by Ali-0111** [comment]
@sameedmohsin28 ,
I added this rule , and thanks for your valuable feedback. It was another chance to code better.

**Review by sameedmohsin28 — APPROVED** [review]
@Ali-0111 

Great mate, mate. Let's merge it.

**Pull request merged**